client/src/services/translation.ts

```diff
@@ -66,10 +84,14 @@ export function initTranslations(locale?: Locale): any {
   return i18n;
 }
 
-export function translate(key: string, attrs?: object, count?: number): string {
+export function msTranslate(content: Translatable): string {
```

Comment:
You can let it be named `translate()`, it shouldn't be exported anymore, everything should be done either with `$msTranslate` (most of the time), or if it can't be done in the template, with inject:
```typescript
const msTranslate = inject('msTranslate');

msTranslate('a.b.c');
```